Create bat script file launching all the microservices.

echo staring mvn install

IF EXIST "pom.xml" (
 # mvn clean install -DskipDocker
  echo installed mvn
) ELSE (
    ECHO failed
)
java -jar .\abixen-platform-eureka\target\abixen-platform-eureka.jar


with the above code mvn builds and disted jars. but somehow  start is failing in windows with below error

Caused by: java.lang.ClassNotFoundException: javax.xml.bind.ValidationException
        at java.base/java.net.URLClassLoader.findClass(Unknown Source) ~[na:na]
        at java.base/java.lang.ClassLoader.loadClass(Unknown Source) ~[na:na]
        at org.springframework.boot.loader.LaunchedURLClassLoader.loadClass(LaunchedURLClassLoader.java:89) ~[abixen-platform-eureka.jar:1.0-SNAPSHOT]
        at java.base/java.lang.ClassLoader.loadClass(Unknown Source) ~[na:na]
        ... 71 common frames omitted

--- a/abixen-platform-run.bat
+++ b/abixen-platform-run.bat
@@ -1,0 +1,26 @@
+echo off
+set var=%1
+
+IF "%var%"=="start" (
+   @echo start maven build
+    call mvn clean package -DskipDocker
+   @echo Maven build completed
+) ELSE (
+    echo Invalid option
+	exit /b 0
+)
+
+@echo starting applications
+call start /b java -jar .\abixen-platform-eureka\target\abixen-platform-eureka.jar
+call start /b java -jar .\abixen-platform-configuration\target\abixen-platform-configuration.jar
+
+echo %time% , sleep for 30 secs
+timeout 30 > NUL
+echo %time% , slept for 30 secs
+
+echo Starting abixen-platform-core, abixen-platform-business-intelligence-service, abixen-platform-web-content-service, abixen-platform-web-client & abixen-platform-zipkin
+call start /b java -jar -Dspring.profiles.active=dev abixen-platform-gateway/target/abixen-platform-gateway.jar
+call start /b java -jar -Dspring.profiles.active=dev abixen-platform-business-intelligence-service/target/abixen-platform-business-intelligence-service.jar
+call start /b java -jar -Dspring.profiles.active=dev abixen-platform-web-content-service/target/abixen-platform-web-content-service.jar
+call start /b java -jar -Dspring.profiles.active=dev abixen-platform-core/target/abixen-platform-core.jar
+call start /b java -jar -Dspring.profiles.active=dev abixen-platform-web-client/target/abixen-platform.war

